packages/testing/containers/n8n-test-container-creation.ts

```diff
@@ -77,12 +79,14 @@ export interface N8NConfig {
 		memory?: number; // in GB
 		cpu?: number; // in cores
 	};
+	proxyServerEnabled?: boolean;
 }
 
 export interface N8NStack {
 	baseUrl: string;
 	stop: () => Promise<void>;
 	containers: StartedTestContainer[];
+	proxyServerUrl?: string;
```

Comment:
Does this need to be exposed or can we get it by going through containers, or maybe a list of URL object instead. Long term if we end up having things like OllamaURL/ProxyURL/LDAP url etc?

Reply:
good point. I managed to refactor it out by getting url from the container itself.